views/attendance/shared_queries.php

```diff
@@ -27,4 +27,8 @@
     "AND emp.employeeid = f.facilitatorid " .
     "AND f.df IS FALSE " .
     "ORDER BY peop.lastname ASC;"
-);
\ No newline at end of file
+);
+
+define("SHARED_QUERY_RACE_ENUM", "SELECT unnest(enum_range(NULL::race));");
```

Comment:
You could always do:
```php
$db->prepare("race_enum", "SELECT unnest(enum_range(NULL::race));");
```

And then whenever you want to use that query you can just do:
```php
$db->execute("race_enum", []);
```

You can also call that `$db->execute()` function as many times as you'd like.

Reply:
if I include the shared_queries files in multiple php files, would this make it so the `$db->prepare` is being called multiple times, or would I have to put it in a class and make an object that's only called once? Also was looking into the parameters of pg_prepare and since nothing has to be parsed in any of the shared queries, I'm not sure if there's much of a difference in speed.

Reply:
if it is in the `shared_queries.php` file, whenever you include it in another PHP file (assuming you're using the `global $db`) it should still work.

Reply:
moved all uses to of shared constant queries to `$db->prepare` and `$db->execute` methods